test/e2e/serial/tests/configuration.go

```diff
@@ -205,10 +205,14 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				Expect(err).ToNot(HaveOccurred())
 
 			}()
-			//so far 0 machine count for mcp-test -> no nodes -> no updates status
+
+			//so far 0 machine count for mcp-test -> no nodes -> no updates status -> empty status
+			var updatedNewMcp *machineconfigv1.MachineConfigPool
+			Expect(fxt.Client.Get(context.TODO(), client.ObjectKeyFromObject(mcp), updatedNewMcp)).To(Succeed())
```

Comment:
`updatedNewMcp` is `nil`, it should be passed as empty initialized struct.
```
updatedNewMcp := &machineconfigv1.MachineConfigPool{}
```

Reply:
right, this was fixed in later commit but should be working starting from this one, fixed. Thanks!